README.md

```diff
@@ -79,4 +79,27 @@ npm run dev
 
 ### Testing
 
+> ❕ We are in process of building automated testing/CI-pipeline. For now all tests needs to be run manually.
+
+Running e2e/component tests with Cypress.
+
+1. Start local server
+```
+npm run dev
+```
+
+2. Open Cypress GUI
+
+```
+npx run cypress
```

Comment:
For some reason, this command doesn't work for me. However, `npx cypress open` does work that opens GUI app
<img width="753" alt="Screenshot 2022-09-22 at 13 49 04" src="https://user-images.githubusercontent.com/31354481/191727818-6d72ce4e-c528-4f00-90dc-260e442161fc.png">

Reply:
Typed wrong command here. My bad. Correct command should be in fact `npx cypress open`